command/src/main/java/pw/chew/jdachewtils/command/OptionHelper.java

```diff
@@ -30,173 +30,328 @@
-     * @param option       The option we want
-     * @param defaultValue The fallback option in case of the absence of the option value
+     * @param key          The option we want
+     * @param defaultValue Nullable default value used in the absence of the option value
      * @return The provided option, or the default value if the option is not present
      */
     @Nullable
     @Contract("_, _, !null -> !null")
-    public static User optUser(@NotNull SlashCommandEvent event, @NotNull String option, @Nullable User defaultValue) {
-        List<OptionMapping> options = event.getOptionsByName(option);
+    public static User optUser(@NotNull SlashCommandEvent event, @NotNull String key, @Nullable User defaultValue) {
+        OptionMapping option = event.getOption(key);
 
-        return options.isEmpty() ? defaultValue : options.get(0).getAsUser();
+        return option == null ? defaultValue : option.getAsUser();
+    }
+    
+    /**
+     * Gets the provided Option Key as a MessageChannel value, or returns {@code null} if the option cannot be found.
+     *
+     * @param event The slash command event to get options from
+     * @param key   The option we want
+     * @return The provided option, or null if the option is not present
+     */
+    @Nullable
+    public static MessageChannel optMessageChannel(@NotNull SlashCommandEvent event, @NotNull String key) {
+        return optMessageChannel(event, key, null);
     }
 
     /**
-     * Guarantees a MessageChannel option value by providing a default value.
+     * Gets the provided Option Key as a MessageChannel value, or returns the default one if the option cannot be found.
      *
      * @param event        The slash command event to get options from
-     * @param option       The option we want
-     * @param defaultValue The fallback option in case of the absence of the option value
+     * @param key          The option we want
+     * @param defaultValue Nullable default value used in the absence of the option value
      * @return The provided option, or the default value if the option is not present
      */
     @Nullable
     @Contract("_, _, !null -> !null")
-    public static MessageChannel optMessageChannel(@NotNull SlashCommandEvent event, @NotNull String option, @Nullable MessageChannel defaultValue) {
-        List<OptionMapping> options = event.getOptionsByName(option);
+    public static MessageChannel optMessageChannel(@NotNull SlashCommandEvent event, @NotNull String key, @Nullable MessageChannel defaultValue) {
+        OptionMapping option = event.getOption(key);
 
-        return options.isEmpty() ? defaultValue : options.get(0).getAsMessageChannel();
+        return option == null ? defaultValue : option.getAsMessageChannel();
     }
 
     /**
-     * Checks to see if the event has an option.
+     * Will return if the provided key resolves into an available Option for the SlashCommand.
```

Comment:
```suggestion
     * Will return if the provided key resolves into a provided Option for the SlashCommand.
```
Should say provided because we only receive what the user input, I think?